history/countries/SBU - Shire Bree Union.txt

```diff
@@ -0,0 +1,24 @@
+# Audax Validator "!" Ignore_1009
+primary_culture = breelander
+culture = breeland_hobbit
+culture = shire_hobbit
+government = democracy
+nationalvalue = generic_leader
+literacy = 0.20
+civilized = yes
+
+oob = "BRE_oob.txt"

```

Comment:
Remove. Don't want someone to accidentally put them on the map and all of a sudden BRE gets extra troops from their OOB getting called twice